criu/files.c

```diff
@@ -1568,12 +1568,22 @@ int inherit_fd_add(int fd, char *key)
 	if (fd > inh_fd_max)
 		inh_fd_max = fd;
 
-	inh->inh_id = key;
+	inh->inh_id = xstrdup(key);
```

Comment:
`xstrdup()` can return NULL. Please add a check if it fails here.

Reply:
Thanks, check added.